### PR TITLE
added email address validation

### PR DESCRIPTION
**Pull request by olaolumcpaul**

### CODE REVIEW

File: src/pages/login/index.js [hunk 1]
```diff
@@ -6,48 +6,87 @@ import CredentialsCard from "../../components/credentials";
...
+              text="Log in"
+              onClick={() => onLogin(formData.email, formData.password)}
+              classes="green width-full"
+            />
+          </form>
+          {/* <Button
+            text="Log in"
+            onClick={() => onLogin(formData.email, formData.password)}
+            classes="green width-full"
+          /> */}
```
Comment (th0jensen) [inline src/pages/login/index.js:85]: Why is this commented out and the button moved into the form?

File: src/pages/login/index.js [hunk 2]
```diff
@@ -6,48 +6,87 @@ import CredentialsCard from "../../components/credentials";
...
+    const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+    return emailPattern.test(emailAddress);
+  };
+
+  const isValidPassword = (password) => {
+    // Example validation: at least 8 characters, 1 uppercase, 1 number, 1 special character
+    const passwordPattern =
+      /^(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+    return passwordPattern.test(password);
+  };
```
Comment (th0jensen) [inline src/pages/login/index.js:23]: Correct me if I'm wrong but aren't the password checks to be added by Atakan in his issue? [Issue #11](https://github.com/boolean-uk/team-dev-client-c13/issues/11)

**Review by th0jensen — COMMENTED** [review]
Left some comments :) Apart from those all LGTM

File: src/pages/login/index.js (hunk 2 above)
Comment (AtikoSpeed) [reply src/pages/login/index.js:23]: Yea

File: src/pages/login/index.js (hunk 1 above)
Comment (olaolumcpaul) [reply src/pages/login/index.js:85]: I moved it into the the form element to make sure it utilises the onSubmit event. Carlo mentioned that there were bugs and i just commented it out to show that it being out of the form element would mean that event handling would need to be done manually.

**Comment by AtikoSpeed** [comment]
Your pull request is in boolean-uk/team-dev-client-template
Move it to boolean-uk/team-dev-client-c13

**Pull request closed without merging**